Documentation/instruction_name_tables.md

```diff
@@ -0,0 +1,62 @@
+## Database table naming rules
+
+Before uploading a new dataset in the database and creating a new table, the following rules should be followed to name the table:
+
+### All names:
+1. Always in lower case; 
+2. No space between the characters. Instead of using space, use `_` to separate composite names.
+3. Column names should follow the above rules.
+
+
+### Schema names:
+
+If the table it's single country data use the full ISO country name. This name for each country of the world can be found [here](https://gadm.org/maps.html).
+
+If the data is on a global scale data use the schema: `global`.
+
+
+### Table names:
+
+Table names should follow this basic structure:
+
+`<locality>_<semantic single word name>_<0|D|W|M>_<optional suffix>`
```

Comment:
I guess we should use the letter in lowercase 

```suggestion
`<locality>_<semantic single word name>_<0|d|w|m>_<optional suffix>`
```

Reply:
I believe an example would be good in here. Good examples are always welcome.